bytecomp/bytegen.ml

```diff
@@ -768,6 +772,11 @@ let rec comp_expr stack_info env exp sz cont =
           Koffsetclosure(pos - env_pos) :: cont
         | exception Not_found -> not_found ()
       end
+  | Lconst (Const_base (Const_float32 f |
```

Comment:
I think this will miss a more complicated constant that contains a `Const_float32` inside it. But perhaps it's impossible to come up with such a constant in bytecode.

Reply:
Good point - const blocks may contain float32s, though `Stdlib_stable` doesn't currently create any. Added logic for it.